### PR TITLE
updated

### PR DESCRIPTION
**Pull request by Durgesh4993**
i will removed all coments. it being right or wrong

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -16,7 +16,6 @@
 
 body{
     width: 500px;
-    /* use dyslexic friendly background */
```
Comment (developer-diganta) [inline css/styles.css:19]: Could you keep this comment? This one helps get an idea of what is happening

**Review by developer-diganta — COMMENTED** [review]
Overall it's great. Just keep that 1 comment

File: css/styles.css (hunk 1 above)
Comment (Durgesh4993) [reply css/styles.css]: Sir thank you for suggesting, completed sir

**Comment by developer-diganta** [comment]
Looks good! Thank You @Durgesh4993! I'll merge it during contribution period!

**Comment by Durgesh4993** [comment]
Ok Thank you sir

**Review by developer-diganta — APPROVED** [review]

**Pull request merged**